spec/js/views/filterViewSpec.js

```diff
@@ -0,0 +1,57 @@
+define([
```

Comment:
Because of the naming change your tests no longer run in karma. 

You can either change karma.conf or stick the naming confention present in the project, snake case for file names.